pkg/cri/instrument/instrumented_service.go

```diff
@@ -29,23 +29,49 @@ import (
 	ctrdutil "github.com/containerd/containerd/pkg/cri/util"
 )
 
+const (
+	// name prefix for CRI sbserver specific spans
+	criSpanPrefix = "pkg.cri.sbserver"
```

Comment:
If both implementations will use the same prefix, should we just use `pkg.cri.server`? Seems like with sbserver moving to the default in 2.0, we don't need to expose `sbserver` in the naming.

Reply:
Fixed